templates/web.ssl.template.yml

```
@@ -15,14 +15,16 @@ run:
      filename: "/etc/nginx/conf.d/discourse.conf"
      from: /listen 80;\s+gzip on;/m
      to: |
-       listen 443 ssl http2;
+       listen 443 ssl;
+       http2;
```

```suggestion
       http2 on;
```

I think this needs to be the above?